src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -445,7 +446,58 @@ protected static void ShiftFieldsFromRootToInstallerLevel(InstallerManifest inst
                     property.SetValue(installerManifest, null);
                 }
             }
-        }
+        }
+
+        /// <summary>
+        /// Shifts common installer fields from installer level to manifest root.
+        /// </summary>
+        /// <param name="installerManifest">Wrapper object containing the installer manifest object models.</param>
+        protected static void ShiftInstallerFieldsToRootLevel(InstallerManifest installerManifest)

```

Comment:
Can you add some tests to verify this behavior? I think a good test case would be to update a manifest that has several installers and verifying that all of the common installer fields get shifted over to the root node.

Reply:
Added a test case to verify the behavior